pyproject.toml

```diff
@@ -24,6 +24,8 @@ tiktoken = "0.7.0"
 uvicorn = "0.23"
 wrapt = "^1.15.0"
 pydantic = "^1.10.12"
+httpx = "^0.27.0"
```

Comment:
Why do we need to add this?
`httpx` is included inside of `fastapi`, do we want this specific version?

Reply:
Because it's an explicit dependency of this project. 
It's literally used somewhere as `import httpx`.